core/src/main/java/org/kiwix/kiwixmobile/core/main/KiwixWebView.kt

```diff
@@ -65,15 +63,6 @@ open class KiwixWebView @SuppressLint("SetJavaScriptEnabled") constructor(
   private fun setWindowVisibility(isFullScreen: Boolean) {
     val window = (context as Activity).window
     WindowCompat.setDecorFitsSystemWindows(window, !isFullScreen)
-    WindowInsetsControllerCompat(window, window.decorView.rootView).apply {
-      if (isFullScreen) {
-        hide(WindowInsetsCompat.Type.statusBars())
-        hide(WindowInsetsCompat.Type.navigationBars())
-      } else {
-        show(WindowInsetsCompat.Type.statusBars())
-        show(WindowInsetsCompat.Type.navigationBars())
-      }
-    }
```

Comment:
@MohitMaliDeveloper Status bar doesn't go away in full screen.

Reply:
hi @gouri-panda 

i have fix it.

Reply:
You reverted it back. but I think we should create a ticket for removing deprecated codes. I think it's important for now because google will soon remove a lot of deprecated classes in upcoming versions.